docs/getting-started.md

```diff
@@ -489,6 +489,18 @@ The _admin token_ is used to edit or delete data within the project. **KEEP THIS
 
 If your lhci server is behind basic auth, you can provide the username and password via the `--basicAuth.password` and `--basicAuth.username` command line flags.
 
+All these options are available for `lhci wizard`
```

Comment:
move and link to the configuration docs instead? don't want to have too many places where every flag is documented extensively :)